invoice/src/test/java/org/killbill/billing/invoice/notification/TestNextBillingDatePoster.java

```diff
@@ -68,7 +65,8 @@ public void testDryRunReInsertion() throws Exception {
         final NotificationQueue nextBillingQueue = notificationQueueService.getNotificationQueue(KILLBILL_SERVICES.INVOICE_SERVICE.getServiceName(),
                                                                                                  DefaultNextBillingDateNotifier.NEXT_BILLING_DATE_NOTIFIER_QUEUE);
         final Iterable<NotificationEventWithMetadata<NextBillingDateNotificationKey>> futureNotifications = nextBillingQueue.getFutureNotificationForSearchKeys(accountRecordId, internalCallContext.getTenantRecordId());
-        final ImmutableList<NotificationEventWithMetadata<NextBillingDateNotificationKey>> futureNotificationsList = ImmutableList.copyOf(futureNotifications);
+        // FIXME-1615: Use import statement
```

Comment:
Same remark as above, it doesn't look like `com.google.common.collect.Iterables` is imported anymore.